website/content/en/guides/level-up/csv-enrichment-guide.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Enrich your observability data 
+short: Enrichment
+description: Learn how to use CSV enrichment to provide more context to your data
+author_github: https://github.com/barieom
+domain: enrichment
+weight: 5
+tags: ["enrichment", "logs", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand the [basic Vector concepts][concepts]
+* You understand [how to set up a basic pipeline][pipeline]
+
+[concepts]: /docs/about/concepts
+[pipeline]: /docs/setup/quickstart
+{{< /requirement >}}
+
+
+It's important for any organization maintaining an observability pipelines to
+have flexibility to manipulate their observability data — whether it be appending
+better context to an event or triggering alerts when there is a potential threat.
+A key component that drives that flexibility is enriching your data from different
+sources. Vector now offers initial support for enriching events from external data
+sources, which is currently powered by a powerful Vector concept,
+[`enrichment tables`][Enrichment tables]. For now, our support for enrichment
```

Comment:
Remove the backticks around enrichment tables. Backticks should only be used for code samples, not terminology.